db/migrate/20170110132223_add_unlimited_participants_to_events.rb

```diff
@@ -0,0 +1,5 @@
+class AddUnlimitedParticipantsToEvents < ActiveRecord::Migration
+  def change
+    add_column :events, :participants_are_unlimited, :boolean, default: false
```

Comment:
This leads to 
` t.boolean  "participants_are_unlimited", default: false`
im my schema.rb file when migrating. But here, the `, default: false` is missing in the `schema.rb`. Is there a reason why? @Baschdl @querenker @aloeser @frederike-ramin